src/services/milestones/milestones.hooks.js

```diff
@@ -117,7 +118,7 @@ const getApprovedKeys = (milestone, data, user) => {
           throw new errors.Forbidden('Only the Milestone Manager can repropose rejected milestone');
         }
         logger.info(`Reproposing rejected milestone with id: ${milestone._id} by: ${user.address}`);
-        return ['status'];
+        return ['status', 'mined', 'message'];
```

Comment:
mined doesn't need to be here, this is a proposed milestone which isn't on chain